src/main/java/org/spongepowered/common/registry/type/world/BlockChangeFlagRegistryModule.java

```diff
@@ -125,7 +125,7 @@ public void registerDefaults() {
             } else if (Flags.OBSERVER == i) {
                 register(new SpongeBlockChangeFlag("OBSERVER".toLowerCase(Locale.ENGLISH), i));
             } else if (Flags.NEIGHBOR_PHSYICS == i) {
```

Comment:
You might as well rename this flag to be spelt correctly while you're at it. As it's in Common only, we don't need to deprecate and remove next version.

Reply:
Good idea, hadn't noticed it.